ci/scripts/sbom_list.py

```
@@ -0,0 +1,120 @@
+        classifiers = info.get("classifiers") or []
+        lic_cls = [c for c in classifiers if c.startswith("License ::")]
+        if lic_cls:
+            return "; ".join(lic_cls)
+        lic = (info.get("license") or "").strip()
+        if lic:
+            return lic
+
+    return "(License not found)"
+
+
+def process_uvlock(uvlock: dict, base_name: str) -> Path:
```

Ruff (0.14.14)

[warning] Unused function argument: `base_name`

(ARG001)